pages/help/help_domain-management.md

```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
 
 Manage DNS information, domain contacts, and more.
  
-- [Manage DNS information for your domain](#)
-- [Add people to your domain account](#)
-- [Add a security email for your domain](#)
-- [Renew your .gov domain](#)
-- [Domain security best practices](#)
+- [Manage DNS information for your domain](#manage-DNS-information-for-your-domain)
+- [Add or update the security email for your domain](#add-or-update-the-security-email-for-your-domain)
+- [Add people to help manage your .gov domain](#add-people-to-help-manage-your-.gov-domain)
+- [Update your contact information (email, phone)](#update-your-contact-information)
```

Comment:
This one isn't working either, should be
`#update-your-contact-information-(email%2C-phone)`

Reply:
Fixed